scripts/convert_GEAR_beam.py

```diff
@@ -0,0 +1,108 @@
+# MJB (UKCEH) Aug-2024
+# Example script for a pangeo-forge-recipe to convert
+# gridded netcdf files to a zarr datastore ready for upload
+# to object storage.
+# See jupyter notebook for more details and explanations/comments
+# Please note that this script/notebook is intended to serve as an example only,
+# and be adapted for your own datasets.
+
+import os
+import pdb
+import apache_beam as beam
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+from apache_beam.options.pipeline_options import PipelineOptions
+from pangeo_forge_recipes.transforms import (
+    OpenWithXarray,
+    StoreToZarr,
+    ConsolidateDimensionCoordinates,
+    ConsolidateMetadata,
+    Indexed,
+    T,    
+)
+
+startyear = 1990
```

Comment:
Even at a prototype stage I'd start where you want to end up and introduce a config file to hold paths and parameters here. 

[omegaconf](https://pypi.org/project/omegaconf/) could be suitable 

[hydra](https://hydra.cc/docs/intro/) - saw this recently being adopted by the [Minerva](https://github.com/Pale-Blue-Dot-97/Minerva/) project which my old team collaborated on. It's not clear to me how one benefits from both!

Either way, a standard-enough, no-thinking-required way of other people managing their configuration separately from your code :D

Reply:
If suggestions are welcome I'd say just `argparse` is probably sufficient for this, and means

1. Fewer dependencies to potentially fight among each other
2. Fewer non-standard tools to distract from the main learning points

Easy to upgrade to something like omegaconf down the line if the need for something more sophisticated arises!